core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -615,6 +615,27 @@ default EventExecutor defaultSubscriberExecutor() {
         return map(function);
     }
 
+    /**
+     * Peeks an error emitted by this {@link StreamMessage} and applies the specified {@link Consumer}.
+     *
+     * <p>For example:<pre>{@code
+     * StreamMessage streamMessage = StreamMessage.aborted(new IllegalStateException("Something went wrong.");
```

Comment:
nit:
```suggestion
     * StreamMessage streamMessage = StreamMessage.aborted(new IllegalStateException("Something went wrong."));
```

Reply:
`StreamMessage<Void>`?

Reply:
Oh~ I missed `)` on `peekError`!
I found that `)` is also missed on `mapError` so fixed it!

> StreamMessage`<Void>`

Thanks for catching 🙇